### DISCUSSION TITLE
Feature Request: Keybase proof integration

### DISCUSSION THREAD

**Discussion by LivInTheLookingGlass** [bookwyrm-social/bookwyrm#1332] [Ideas]
It would be neat if we could prove identities on BookWyrm instances, like you can on Mastodon instances. This is a way to prove that you are who you say you are in a more decentralized way.

**Comment by hughrun** [comment]
Mastodon does have some sort of keybase integration I believe, but also implements [`rel="me"`](http://microformats.org/wiki/rel-me) which is a simpler and more decentralised way to prove identity or at least control of a web entity.

**Reply by jokeyrhyme** [reply]
I've been using [Keyoxide](https://keyoxide.org/) now that the open future of Keybase is less certain (and Mastodon has already dropped Keybase-specific support in favour of generic links on our profile pages that are perfect for `[rel=me]` usage
